homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -187,9 +189,23 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 
     # Create handler
     _LOGGER.info("Initializing with host %s (token %s...)", host, token[:5])
-    vacuum = Vacuum(host, token)
 
-    mirobo = MiroboVacuum(name, vacuum)
+    unique_id = None
+
+    try:
+        vacuum = Vacuum(host, token)
```

Comment:
Only wrap the line where an error could occur.

Reply:
That would be the `info` call in this case.

Reply:
This should be cleaned up in the other platforms as well when changes discussed in https://github.com/home-assistant/core/pull/32091#issuecomment-612985685 are implemented. They are all inconsistent.